apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryClientBuilder.cs

```diff
@@ -66,6 +88,7 @@ public override async Task<BigQueryClient> BuildAsync(CancellationToken cancella
         protected override void Validate()
         {
             base.Validate();
+            // TODO: Validate that if Service is set, no credentials are set?
```

Comment:
That would be inline with what we do with the invokers in the gRPC equivalents. Can't Service here just be internal? We can always make it public later on if there's a need for it apart from testing.

Reply:
We could definitely make it internal, yes - I just don't like the idea that *we* can test stuff but others can't. Still, it means we don't need to worry about the TODO for now. Will remove this TODO, but add a note on the newly-internal property.